File: ruby/day7/largest.rb
arr = [1,2,3,3,345,345,234,52,345,234,534,54,3]

def largest(*numbers)
  numbers.max
end

puts largest 6,23,4,2
puts arr.max
puts largest 1, 2, 3,3 ,345,345,234,52,345,234,534,54,3
puts largest 3, 9, 13
puts largest 3, 5

File: Internet_Protocol/day17/Labs/largest_number/largest.js
// let numArray = [parseInt(firstNum), parseInt(secondNum), parseInt(thirdNum)];

var str = '3 , 6, 2, 56, 32, 5, 89, 32';

var arr = str.split(',');

console.log(arr);
// var array = [3 , 6, 2, 56, 32, 5, 89, 32];

     //console.log(typeof(numArray[0]));
     let largest = 0;
     for(let i = 0; i < arr.length; i++){
       if(arr[i] > largest) {
         largest = arr[i];
       }
     }

  console.log(largest);
